### ISSUE TITLE
time of occurrences in download

### ISSUE THREAD

**Issue by chrisvanswaay** [BiologicalRecordsCentre/ABLE#466]
@DavidRoy @kazlauskis @CrisSevilleja 
Something else surprised me in the occurrences download, where all occurrences in a sample have the same starttine and endtime as the sample. Now I understand this was the case in the first versions of the app (and nothing wrong with copying it to the occurrences, though not needed as already in the sample download), but nowadays in the app I can see all occurrences are logged with the exact time. Is it possible to make these exact times available in the occurrences download (as far as they have been collected, i.e. in the recent versions of the app)?

**Comment by DavidRoy** [comment]
I agree that each occurrence should have it's own timestamp (not start/end time).
@kazlauskis @Vilius-Stankaitis can you implement this if not already done - it may be that this information is in the database but not in the downloads

**Comment by Vilius-Stankaitis** [comment]
it should be in the database as occAttr:912 (occurrence time of sighting).

**Comment by JimBacon** [comment]
I have added the Time of Sighting attribute to the 15-minute downloads.

It appears to be using a different timezone to the start and end time attributes which is not very nice. I think there is another piece of work to be done here.
